pkg/doc/verifiable/doc.go

```diff
@@ -0,0 +1,15 @@
+/*
```

Comment:
Is this file supposed to be empty? Doesn't look like there's anything actually in here

Reply:
Nevermind - I looked over the commented part too quickly. Does this instead belong as a .md file in the docs folder?

Reply:
This file is not empty - it is providing the package comment for GoDoc. Note that the package comment can actually be in any go file (doesn’t need to be a special doc.go).

Reply:
e.g., the existing common.go file.

Reply:
OK! I will change this at next PR!

Reply:
No, I will do it now, just noticed the PR is not merged yet.

Reply:
Moved to `common.go`. Thanks @troyronda @DRK3  !